stride/chain.json

```diff
@@ -0,0 +1,135 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "stride",
+    "status": "live",
+    "network_type": "mainnet",
+    "pretty_name": "Stride",
+    "chain_id": "stride-1",
+    "bech32_prefix": "stride",
+    "daemon_name": "strided",
+    "node_home": "$HOME/.strided",
+    "genesis": {
+        "genesis_url": "https://raw.githubusercontent.com/Stride-Labs/testnet/infra-test/poolparty/infra/genesis.json"
+    },
+    "key_algos": [
+        "secp256k1"
```

Comment:
is this the right algo? I thought we were using ed25519

Reply:
good catch, replaced with ed25519